applications/openshift/general/version_detect_in_hypershift/rule.yml

```diff
@@ -5,7 +5,7 @@ prodtype: ocp4
 title: 'This is a helper rule to fetch the required api resource for detecting HyperShift OCP version'
 
 {{% set custom_jqfilter = '[.status.version.history[].version]' %}}
-{{% set custom_api_path = '/apis/hypershift.openshift.io/v1alpha1/namespaces/clusters/hostedclusters/{{.hypershift_cluster}}' %}}
+{{% set custom_api_path = '/apis/hypershift.openshift.io/v1beta1/namespaces/clusters/hostedclusters/{{.hypershift_cluster}}' %}}
```

Comment:
What happens if we go to use this content on an older OCP release?

Reply:
it depends on what hypershift they installed, but that shouldn't cause an issue here. Because if the fetched hypershift version resource does not exist, it will just use ocp version

Reply:
After talking through this with Vincent, this should be OK. The alpha channel is deprecated and will be removed.

We haven't released hypershift support yet so we should be ok to update this.